scripts/wsrep_sst_clone.sh

```diff
@@ -328,98 +299,160 @@ setup_clone_plugin()
+        wsrep_log_info "CLONE SSL variables and SSL client are not correctly set: @@clone_ssl_cert='$CLONE_SSL_CERT', @@clone_ssl_key='$CLONE_SSL_KEY'"
+    fi
+
+    # Checking Server certificates
+    # Check that we have all the files
+    # If they have not been explicitly specified, check the datadir
+    if [ -z "$SERVER_SSL_CA" ]; then
+        if [ -r "$DATA/ca.pem" ]; then
+            SERVER_SSL_CA="$DATA/ca.pem"
         else
-            wsrep_log_error "CLONE SSL variables are explicitly empty: @@clone_ssl_cert='$CLONE_SSL_CERT', @@clone_ssl_key='$CLONE_SSL_KEY'"
-            wsrep_log_error "This means Clone plugin was already present but clone_ssl_ variables are not set. This is an error, variables must be set before continue."
-            exit 1
+            wsrep_log_error "******************* FATAL ERROR ********************** "
+            wsrep_log_error "* Could not find a CA (Certificate Authority) file.  "
+            wsrep_log_error "* Please specify a CA file with the 'ssl-ca' option. "
+            wsrep_log_error "* Line $LINENO"
+            wsrep_log_error "**************************************************** "
+            return 2
         fi
-        CLEANUP_CLONE_SSL=
+    fi
+    if [ -z "$SERVER_SSL_CERT" ]; then
+        if [ -r "$DATA/server-cert.pem" ]; then
+            SERVER_SSL_CERT="$DATA/server-cert.pem"
+        else
+            wsrep_log_error "******************* FATAL ERROR ********************** "
+            wsrep_log_error "* Could not find a certificate file.                            "
+            wsrep_log_error "* Please specify a certificate file with the 'ssl-cert' option. "
+            wsrep_log_error "* Line $LINENO"
+            wsrep_log_error "****************************************************** "
+            return 2
+        fi
+    fi
+    if [ -z "$SERVER_SSL_KEY" ]; then
+        if [ -r "$DATA/server-key.pem" ]; then
+            SERVER_SSL_KEY="$DATA/server-key.pem"
+        else
+            wsrep_log_error "******************* FATAL ERROR ********************** "
+            wsrep_log_error "* Could not find a key file.                           "
+            wsrep_log_error "* Please specify a key file with the 'ssl-key' option. "
+            wsrep_log_error "* Line $LINENO"
+            wsrep_log_error "****************************************************** "
+            return 2
+        fi
+    fi
+    if [ -n "$SERVER_SSL_CA" ] && [ -n "$SERVER_SSL_CERT" ] && [ -n "$SERVER_SSL_KEY" ]
+    then
+        SERVER_SSL_VALID="yes"
+    else
+        SERVER_SSL_VALID="no"
     fi
 
+    # We reset the variable to pass ssl information and will fill it only if both SERVER and CLIENT 
     CLIENT_SSL_OPTIONS=""
-    if [ -n "$CLONE_SSL_CERT" -a -n "$CLONE_SSL_KEY" ]
+    wsrep_log_debug ":->(POST) CLIENT_SSL_VALID=$CLIENT_SSL_VALID SERVER_SSL_VALID=$SERVER_SSL_VALID"
+    if [ "$CLIENT_SSL_VALID" == "yes" ] && [ "$SERVER_SSL_VALID" == "yes" ]
```

Comment:
As we discussed, everything what happens here, affects only donor, so should happen only on donor (the result is setting 
1. `CLIENT_SSL_OPTIONS` - used during Donor -> clone instance connection
2. `REQUIRE_SSL` - creation of clone user on Donro server.
These are two independent things

Reply:
done